### DISCUSSION TITLE
RuntimeError: "addmm_impl_cpu_" not implemented for 'Half'

### DISCUSSION THREAD

**Discussion by hodgesz** [WeOpenML/PandaLM#27] [Q&A] [answered]
Apologies to be the only one asking questions, but we love the project and think it will really help us in evaluating different LLMs for our use cases.

We moved out of Colab to make things easier to debug. We are now trying to install on our own hardware - server w/ 3070 desktop card and 64 GB of RAM.

We are now able to launch the Chat UI without a problem.

![image (12)](https://github.com/WeOpenML/PandaLM/assets/1764965/4aee5962-3651-4092-8176-071614718901)

However, we run into the following RuntimeError from there.

![image (10)](https://github.com/WeOpenML/PandaLM/assets/1764965/b023919a-1a6f-49e5-97fc-69497562e9ca)

Any thoughts?

**Comment by zhuohaoyu** [comment, marked as answer]
Similar issues can be found on StackOverflow, I think your model is loaded on CPU instead of your GPU.  https://stackoverflow.com/questions/73530569/pytorch-matmul-runtimeerror-addmm-impl-cpu-not-implemented-for-half

**Reply by hodgesz** [reply]
Thanks, will take a look there.
